tools/aws_lab_setup/roles/student_users/templates/student_list.j2

```diff
@@ -0,0 +1,6 @@
+users:
+{% for student in range(student_total) %}
+  - name: student{{ loop.index }}
+    username: student{{ loop.index }}
+    email: {{instructor_email | default('dummy_email@redhat.com')}}
```

Comment:
Cant use redhat.com here. If you need a "dummy" email address might I suggest something even more generic like no-reply@example.com?